CONTRIBUTING.md

```diff
@@ -26,15 +27,15 @@ project's dependencies:
 npm install
 ```
 
-The general flow will often start with adding a test case to `./corpus`. You can
+The general flow will often start with adding a test case to `./test/corpus`. You can
 find details on how testing works with tree-sitter
-[here](https://tree-sitter.github.io/tree-sitter/creating-parsers#command-test).
+[here](https://tree-sitter.github.io/tree-sitter/creating-parsers/5-writing-tests.html).
 
 Once you've added your test case you'll want to then make the required changes
 to `grammar.js`, regenerate and recompile the parser, and run the tests:
 
 ```sh
-npm run build
+tree-sitter generate
```

Comment:
Is this the command to use now? I see it is used in CI (https://github.com/tree-sitter/tree-sitter-scala/blob/master/.github/workflows/sync.yml#L27), but I’m not sure if anything else also needs to be called?

Reply:
I think the main testing CI script is https://github.com/tree-sitter/parser-test-action/blob/master/action.yml, and looks like it should be

```bash
tree-sitter generate
tree-sitter test
```

I think.